app/containers/Forms/FuelField.tsx

```diff
@@ -26,17 +33,32 @@ const FuelField: React.FunctionComponent<Props> = (props) => {
     else handlefuelChange(fuel.fuelRowId);
   };
 
-  return <ObjectField {...props} onChange={handleChange} />;
+  return (
+    <>
+      {isFuelArchived && (
+        <Alert variant="danger">
+          <strong>Warning:</strong> This fuel has been archived. Please remove
+          it and select an appropriate alternative.
+        </Alert>
```

Comment:
Will this alert be displayed to the reviewer when looking at an existing application? If so it might be confusing when they look at 2019 applications?

Reply:
Yes, they show up on the summary page as well, I replicated the behaviour and look that was used for archived products. It could be a low impact improvement to clean this up, maybe a tech debt card?

Reply:
It can be cleaned up later, but it's not tech tebt, it's a feature